java/beans/src/org/netbeans/modules/beans/PropertyPatternPanel.java

```diff
@@ -519,7 +519,7 @@ public void actionPerformed( java.awt.event.ActionEvent e ) {
         if (TYPES == null) {
             String typeList = NbBundle.getMessage(PropertyPatternPanel.class, "IdxPropertyPatternPanel_Types"); // NOI18N
             StringTokenizer st = new StringTokenizer(typeList, "|"); // NOI18N
-            List l = new LinkedList();
+            List<String> l = new LinkedList();
```

Comment:
The instanziation is missing a diamond operator.

Reply:
good eyes.. fixed..

Reply:
The list still accepts any objects. Is that wanted?

Reply:
Ugg.. Mismerge on my part.. Give me a second to fix.. Sorry about that..

Reply:
Should be fixed now..